src/tests/lib/qunit_config.js

```diff
@@ -1 +1,41 @@
-QUnit.config.testTimeout = 6400;
+/* globals badger:false */
```

Comment:
Could this be moved to `src/tests/qunit_config.js` or `src/tests/tests/qunit_config.js`? When I see a "`lib`" I unusually think of it as only containing code from a third party.